help/dns_verification/dns_verification.rst

```diff
@@ -0,0 +1,87 @@
+.. _doc_dnsverification:
+.. _Dashboard: https://mailsac.com/dashboard
+.. _Domains: https://mailsac.com/domains
+
+DNS Verification
+================
+
+DNS (Domain Name System) is at the core of the internet and email delivery.
+When configuring a :ref:`BYO Domain <section_byod_configuration>` the following
+records need to be configured:
```

Comment:
The MX records are optional, so not "need to be configured"

The verification of domain ownership only requires the DKIM and SPF records.